fe/fe-core/src/main/java/org/apache/doris/binlog/BinlogManager.java

```diff
@@ -446,6 +447,17 @@ public void addDropRollup(DropInfo info, long commitSeq) {
         addBinlog(dbId, tableIds, commitSeq, timestamp, type, data, false, info);
     }
 
+    public void addRecoverTableRecord(RecoverInfo info, long commitSeq) {
```

Comment:
The `RecoverInfo` comes from the old version of Doris, which does not contain `TableName` and `PartitionName`, and should be filtered here. Otherwise, it might cause the ccr syncer to stop synching.

Reply:
thank you. have update it.